src/elements/content-uploader/__tests__/ItemRemove.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import noop from 'lodash/noop';
+
+import PlainButton from '../../../components/plain-button';
+import { ItemRemoveBase as ItemRemove } from '../ItemRemove';
+
+import { STATUS_IN_PROGRESS } from '../../../constants';
+
+describe('elements/content-uploader/ItemRemove', () => {
+    test('should have aria-label "Remove" and no aria-describedby', () => {
+        const wrapper = shallow(
+            <ItemRemove
+                intl={{ formatMessage: data => data.defaultMessage }}
+                isResumableUploadsEnabled
+                onClick={noop}
+                status={STATUS_IN_PROGRESS} // Use any status. The label does not change based on status.
```

Comment:
feel free to keep this but it's also okay to omit comments if they could be inferred from the component logic

Reply:
i did this for me - i am less than stellar at inferring logic :sadparrot: